app/models/manager_refresh/save_collection/base.rb

```diff
@@ -2,16 +2,16 @@ module ManagerRefresh::SaveCollection
   class Base
     class << self
       def save_inventory_object_inventory(ems, inventory_collection)
-        _log.info("Synchronizing #{ems.name} collection #{inventory_collection} of size #{inventory_collection.size} to"\
-                " the database")
+        _log.info("SYNCHRONIZING collection #{inventory_collection} of size #{inventory_collection.size} to"\
```

Comment:
Why are you yelling? :)  Seriously though is the all caps needed?  And if so for consistency shouldn't the one below be all caps as well?

Reply:
done